static/index.html

```diff
@@ -183,11 +183,11 @@
       return stash("<input type=text id=search onInput='get()' onKeyPress='got()'><br><div id=searchresult></div>")
     }
     function fullsearch () {
-      return stash("<form action=\"http://c2.com/cgi/fullSearch\"><input type=text name=search></form>")
+      return stash("<form action=\"https://c2.com/cgi/fullSearch\"><input type=text name=search></form>")
     }
     var prepass = text
       .replace(/〖(\d+)〗/g, '〖 $1 〗')
-      .replace(/^http:\/\/(www.)?youtube.com\/watch\?v=([-\w]+)/, youtube)
```

Comment:
Looking at this more closely, if this is processing data from the text of the articles I'm assuming that the regex should match on `http` and `https` here then perform the replace. Similarly for `external` links below?